chromium_src/chrome/browser/privacy_sandbox/browsing_topics_settings_interactive_uitest.cc

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include <stddef.h>
+
+namespace chrome {
+inline constexpr char kPrivacySandboxManageTopicsURL[] =
+    "chrome://settings/adPrivacy/interests/manage";
+inline constexpr char kPrivacySandboxAdTopicsURL[] =
+    "chrome://settings/adPrivacy/interests";
+}  // namespace chrome
+
+#define StartWithTwoBlockedTopics DISABLED_StartWithTwoBlockedTopics
+#define BlockFirstTopicOnManageTopicsPage \
+  DISABLED_BlockFirstTopicOnManageTopicsPage
+#define UnblockOneTopicOnAdTopicsPage DISABLED_UnblockOneTopicOnAdTopicsPage
+#define ConfirmDefaultIconIsNotUsed DISABLED_ConfirmDefaultIconIsNotUsed
```

Comment:
sorry, I don't see how these defines are related to tests build.

we have `*.filter` files to disable upstream tests, so we shouldn't disable them via defines here.

Reply:
Since the `kPrivacySandboxManageTopicsURL` and `kPrivacySandboxAdTopicsURL` pages are not available in Brave, and those added by me are just for fixing the build, I wanted to disable these tests. Disabling the tests via a config file is much better. Thanks!

Reply:
pls remove these defines, let's only fix the build with your PR. I'd like to merge it, but this is not something we should have here.

Reply:
@goodov Done. Thanks!